.github/workflows/pr.yml

```
@@ -49,6 +16,7 @@ jobs:
         uses: actions/setup-node@v1
```

```suggestion
        uses: actions/setup-node@v2.4.0
```

`actions/setup-node` is stable at `@v2.4.0` now: https://github.com/actions/setup-node/releases

We don't get any GitHub Action updates from Renovate and they are not pinned like on our main repo. So we'll need to check why that is...

---

Is Renovate even enabled for this repository?

I see we have a Renovate config file in the repository here: https://github.com/renovatebot/parser-utils/blob/main/.github/renovate.json but that config looks nothing like the one we use on the main `renovatebot/renovate` repository.

---

Hopefully, the cache can be reused for `trunk` too, I'll push one more change for this